frontend/pages/ManageControlsPage/SetupExperience/cards/BootstrapPackage/components/BootstrapPackageTable/BootstrapPackageTableConfig.tsx

```diff
@@ -82,21 +82,31 @@ export const COLUMN_CONFIGS: IColumnConfig[] = [
       />
     ),
     accessor: "hosts",
-    Cell: ({
-      cell: { value: aggregateCount },
-      row: { original },
-    }: ICellProps) => {
+    Cell: ({ cell: { value: aggregateCount } }: ICellProps) => {
       return (
-        <div className="bootstrap-package-table__aggregate-table-data">
-          <TextCell value={aggregateCount} formatter={(val) => <>{val}</>} />
-          <ViewAllHostsLink
-            className="view-hosts-link"
-            queryParams={{
-              bootstrap_package: original.status.value,
-              team_id: original.teamId,
-            }}
-          />
-        </div>
+        <TextCell value={aggregateCount} formatter={(val) => <>{val}</>} />
+      );
```

Comment:
Refactored all "View all hosts" links to follow the same pattern (Having a separate column)